W-14373687 recorder 6.5.0 release (#312)
mulesoft/docs-rpa: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-recorder-6.5.0-release-notes.adoc

= RPA Recorder 6.5.0 Release Notes

RPA Recorder enables you to record a business process for automation.

include::partial$release-notes/rn-last-version-message.adoc[] 

== November 28, 2023

// == What's New
include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]

> * The *Retake* option now displays as a button in a dropdown. Using the *Retake* option, you can delay the screen freeze for capturing image-based mouse clicks. This assists with preparing collapsible menus and hidden elements before the screen capture freeze. User and mouse actions are not recorded during the delay.

Reviewer: ```suggestion
* The *Retake* option is now shown as a button in a dropdown. Using the *Retake* option, you can delay the screen freeze for capturing image-based mouse clicks. This assists with preparing collapsible menus and hidden elements before the screen capture freezes. User and mouse actions are not recorded during the delay.
```

* You can now set the initial screen freeze delay directly from the image-based mouse clicks toolbar.
* When recording a Web Task, the selected area now has a red border.
* RPA Recorder now always generates an `RPARecorder` subfolder during installation. 
* RPA Recorder now uses TLSv1.3 by default, and HTTP is now disabled for all API requests. 
* When recording in Notepad++, long window titles and name attributes are now truncated in the Documentation section of the RPA Recording Editor. 

> * Updated the UI of the Settings and Log files panels, as well as the Manual Activity and Comment dialogs.

Reviewer: ```suggestion
* The Settings UI and Log files panels are updated, as well as the Manual Activity and Comment dialogs.
```

== Other Important Changes

> * The attribute selection panel is no longer enabled and cannot be expanded or collapsed when recording image-based mouse clicks in a Web Task. Once image-based mouse clicks recording is finished, the panel can be opened.

Reviewer: ```suggestion
* The attribute selection panel is no longer enabled and cannot be expanded or collapsed when recording image-based mouse clicks in a Web Task. After the image-based mouse clicks recording is finished, the panel can be opened.
```

|When capturing image-based mouse clicks during Web Task recording, the browser window now freezes properly when selecting a dropdown menu. |W-14036847
|When capturing image-based mouse clicks, pressing _Alt+F4_ in the capture image step no longer leads to an error. |W-14025390
|When capturing image-based mouse clicks, the elapsed time now matches the specified delay time. |W-14007202
|Changing the scaling when capturing image-based mouse clicks is now recognized. |W-13778082
|RPA Recorder no longer fails to load the BPMN from RPA Manager when the width or height values are not a whole number. |W-13957382
|RPA Recorder no longer crashes when attempting to open a web page that has an invalid URL containing two periods. Instead, a `Page is not available` error message displays. |W-13990958
|When recording a hover event for a Bot Task, the active window is now recorded. |W-12752823
|When opening a recorded BPMN in Notepad++ in a Bot Task, HTML is now being escaped in documentation properly and only the events from the recording are documented. |W-14292760

> |An error message now displays when a BPMN fails to download. |W-14128004

Reviewer: ```suggestion
|An error message now appears when a BPMN fails to download. |W-14128004
```
